gbpservice/nfp/orchestrator/drivers/orchestration_driver_base.py

```diff
@@ -212,6 +281,10 @@ def get_network_function_device_sharing_info(self, device_data):
         ):
             raise exceptions.IncompleteData()
 
+        self._update_vendor_data(device_data['service_details']['image_name'],
```

Comment:
Do we need to call this in multiple functions? Isn't the meta data same for a particular image?